### PR TITLE
Remove JSX/React

### PR DESCRIPTION
**Pull request by jstavh**
Removed JSX and React from the entire codebase.

### CODE REVIEW

**Comment by jstavh** [comment]
@robinbb see here https://github.com/Granola-Team/site/issues/53 also since for our JSX files to render we need to use @astrojs/react, @astrojs/preact, or @astrojs/solid-js according to https://docs.astro.build/en/reference/errors/no-matching-renderer/ so please let me know which one you want.

**Comment by robinbb** [comment]
@jstavh Can we get rid of JSX?

**Comment by jstavh** [comment]
> @jstavh Can we get rid of JSX?

@robinbb -- I removed JSX/React.

File: src/astro/package.json [hunk 1]
```diff
@@ -11,15 +11,10 @@
     "build-css": "tailwindcss build -i src/styles/styles.css -o public/styles/styles.css"
   },
   "dependencies": {
-    "@astrojs/mdx": "^1.1.1",
-    "@astrojs/react": "^3.0.3",
+    "@astrojs/mdx": "^1.0.3",
```
Comment (robinbb) [inline src/astro/package.json]: You are inadvertently downgrading mdx, here, I think. Run 'git fetch && git merge origin/main' while on this branch.

File: src/astro/package.json [hunk 2]
```diff
@@ -11,15 +11,10 @@
...
     "@astrojs/rss": "^3.0.0",
-    "@astrojs/tailwind": "^5.0.2",
-    "@types/react": "^18.2.28",
-    "@types/react-dom": "^18.2.13",
-    "astro": "^3.3.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "@astrojs/tailwind": "^5.0.0",
+    "astro": "^3.2.4",
```
Comment (robinbb) [inline src/astro/package.json]: Same for astro and tailwind.

**Review by robinbb — CHANGES_REQUESTED** [review]

**Comment by robinbb** [comment]
Holy shit! This is amazing!

**Comment by jstavh** [comment]
> Holy shit! This is amazing!

@robinbb -- Thank you! I'm happy you like the improvements!

File: src/astro/package.json (hunk 1 above)
Comment (jstavh) [reply src/astro/package.json]: Good catch. Corrected this in the next commit.

File: src/astro/package.json (hunk 2 above)
Comment (jstavh) [reply src/astro/package.json]: Good catch. Corrected this in the next commit. See above.

**Review by robinbb — APPROVED** [review]

**Pull request merged**